pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -82,58 +82,20 @@ func (e *InvalidRegistries) Error() string {
 }
 
 // parseURL parses the input string, performs some sanity checks and returns
-// the sanitized input string.  An error is returned in case parsing fails or
-// or if URI scheme or user is set.
+// the sanitized input string.  An error is returned if the input string is
+// empty or if contains an "http{s,}://" prefix.
 func parseURL(input string) (string, error) {
 	trimmed := strings.TrimRight(input, "/")
 
 	if trimmed == "" {
 		return "", &InvalidRegistries{s: "invalid URL: cannot be empty"}
 	}
 
-	// Ultimately, we expect input of the form example.com[/namespace/…], a prefix
```

Comment:
Fair enough, I guess we need this relaxed urgently enough that waiting for a correct parser is not reasonable.

Reply:
Yes, that's unfortunate but necessary. Moby doesn't hit the issue as it enforces the URI scheme to either be `http` or `https`.